### PR TITLE
Move snapshot error msg to error pkg

### PR DESCRIPTION
**Pull request by Madhu-1**
Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

File: glusterd2/commands/snapshot/snapshot-deactivate.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/gluster/glusterd2/glusterd2/transaction"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/errors"
```
Comment (prashanthpai) [inline glusterd2/commands/snapshot/snapshot-deactivate.go]: Needs a blank line after glusterd2 package imports.

Comment (Madhu-1) [reply glusterd2/commands/snapshot/snapshot-deactivate.go]: done

**Comment by Madhu-1** [comment]
@aravindavk @rafikc30 PTAL

**Comment by Madhu-1** [comment]
@aravindavk PTAL

**Pull request merged**